offline_imu_cam/plugin.cpp

```diff
@@ -54,6 +81,14 @@ class offline_imu_cam : public ILLIXR::threadloop {
 			? std::make_optional<cv::Mat*>(sensor_datum.cam1.value().load().release())
 			: std::nullopt
 			;
+		camera_cvtfmt_log.log(record{__camera_cvtfmt_header, {
```

Comment:
This is no longer timing camera conversion, so please rename appropriately.

Rest looks good.